.travis.yml

```diff
@@ -25,9 +25,8 @@ matrix:
       dist: trusty
       sudo: required
       services: docker
-      env: DOCKER_IMAGE=ubuntu:16.04 SWIFT_SNAPSHOT=4.0
```

Comment:
Could you put the DOCKER_IMAGE env variable back? We still need to test 16.04.

Reply:
@sandmman @ShihabMehboob made the requested change. The travis builds finally completed. I will process the PR now, thanks!